packages/kubernetes-1.18/kubelet-config

```diff
@@ -29,6 +29,8 @@ authorization:
 clusterDomain: {{settings.kubernetes.cluster-domain}}
 clusterDNS:
 - {{settings.kubernetes.cluster-dns-ip}}
+evictionHard:
```

Comment:
what would happen if user does not set this setting ? I guess this setting would still exist in kubelet config without any value. Should we wrap setting in if ? for example:
```
{{~#if settings.kubernetes.evictionHard }}
evictionHard:
      {{join_map ": " "\n  " "no-fail-if-missing" settings.kubernetes.eviction-hard}}
{{~/if}}
```
I would let others weigh in, I am also learning in this commit :)

Reply:
Correct. The `eviction-hard` setting would still there and would not effect `kubelet start`. Your example is a good suggestion to me. Thanks 👍

Reply:
Yes - I think we should wrap this in an `if`.